python/functional-programming/functional-programming/lambda-functions.md

```diff
@@ -50,6 +50,21 @@ In this case, we are passing the lambda function `x % 3 == 0` to `filter()` to b
 
 Note: Lambda should only be used when a function will be used in one location in your code, and won't be used again after that. In all other cases, it is better to use `def` to avoid unnecessary confusion.
 
+---
+## Practice
+
+Filter the following list such that all elements are > 0. Use lambda functions:
+
+```python
+
+l = [-1,-2,3,-4,5,6]
+
+new = filter(??? x: ???, ???)
+```
+* lambda
+* x > 0
+* l
```

Comment:
add some wrong answers here

Reply:
I added more answers here, it is just github that doesn't compress this comment.